R/ard_crosstab.R

```diff
@@ -1,46 +1,96 @@
-#' ARD Crosstab
+#' Cross Tabulation ARD
 #'
-#' Similar to `ard_categorical()` except the percentages and N statistics are
-#' based on the entire data set, rather than stratified columns.
+#' Produces summary statistics similar to those that would appear in a
+#' cross tabulation of two variables.
 #'
 #' @inheritParams ard_categorical
 #' @param by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
-#'   columns to compute statistics by.
+#'   a single column to compute statistics by.
+#' @param percent (`string`)\cr
+#'   one of `c("cell", "column", "row")` that determines how the percentage is
```

Comment:
2 comments here: 
- Did you consider allowing the user to pass multiple choices to `percent` instead of requiring separate calls, especially if the individual calls can also be achieved with `ard_categorical`?
- Would it be possible to add a bit more clarification about how the user should think about "row" vs "column" e.g. "row" calculates percentages across the levels of `by`, vs "column" computes percentages within `by` across the levels of `variables`

Reply:
> * Did you consider allowing the user to pass multiple choices to `percent` instead of requiring separate calls, especially if the individual calls can also be achieved with `ard_categorical`?

If there are tables that use combinations of these, then we can consider it. But if not, I think it's ok to leave as is. I can't think of a table that mixes these...

> * Would it be possible to add a bit more clarification about how the user should think about "row" vs "column" e.g. "row" calculates percentages across the levels of `by`, vs "column" computes percentages within `by` across the levels of `variables`

Ahh good point. I'll update!